frontend/packages/ux-editor/src/components/config/editModal/EditOptions/OptionTabs/EditOptionList/EditOptionList.tsx

```diff
@@ -101,6 +101,8 @@ export function EditOptionList<T extends SelectionComponentType>({
   );
 }
 
+const isUnknownError = (error: AxiosError<ApiError>) => !error.response?.data?.errorCode;
```

Comment:
You're both right in some way. Ideally, to conform with our internal coding principles, we should create a whole new file for this function, since it is not dependent on React. That way, different concepts will be clearly separated from each other, and the newspaper article principle will no longer be an issue.